src/components/trash.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import annotations
+
+import disnake
+
+from src.constants import Emoji
+
+TRASH_BUTTON_CUSTOM_ID = "TRASH:"
+
+
+class TrashButton(disnake.ui.Button[None]):
+    """A button that will delete the message when clicked.
+
+    This button uses low-level component listeners (@bot.listen('on_button_click')
```

Comment:
Button classes cannot use or not use LLCs by themselves, only the caller decides

Reply:
Should be fixed now.